src/Cache/FileCache.php

```diff
@@ -190,6 +190,19 @@ public function onRequest(RequestEvent $requestEvent, string $eventName, EventDi
             $cacheFilePath = $this->getCacheFilePath($request, $nativeImage);
             $cacheFile = new File($cacheFilePath, false);
             $firstGen = false;
+
+            /**
+             * Also check date, if cached date is lower then original date -> Remove cached file
+             */
+            $mtime_original_file = $nativeImage->getRequestedFile()->getMTime();
+            $mtime_cached_file = $cacheFile->getMTime();
+            if ($mtime_original_file !== false && $mtime_cached_file !== false) {
+                if ($mtime_cached_file < $mtime_original_file) {
```

Comment:
Thanks for your update @indual-emanuel 

Can you factorize your 2 conditions in one please?